### DISCUSSION TITLE
Attaching a MetricReader to an existing OpenTelemetry instance

### DISCUSSION THREAD

**Discussion by KevinLiLu** [open-telemetry/opentelemetry-java#6274] [Q&A]
Is it possible to attach a `MetricReader` (ex. the `InMemoryMetricReader`) to an existing `OpenTelemetry` instance which has already been created (ex. `GlobalOpenTelemetry`)? We want to programmatically fetch metrics emitted by a framework which is using the global instance, however we don't have control over the creation of the global instance.

**Comment by jkwatson** [comment]
This is not currently possible. The configuration of the OpenTelemetry instance is immutable.
